corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -102,17 +110,29 @@ hqDefine("geospatial/js/case_grouping_map",[
             hiddenElement.remove();
         };
 
-        self.addGroupsToCases = function(caseGroups) {
-            clearCaseGroups();
+        self.addGroupDataToCases = function(caseGroups, groupsData, assignDefaultGroup) {
+            const defaultGroup = groupsData[0];
```

Comment:
are we assuming there that the first group is the unassigned one?

Reply:
@mkangia Correct.

Reply:
is there a more reliable way to do that instead of using index?

Reply:
Yeah, I suppose so. I haven't given it much attention after I wrote it, but maybe I should. I'll update.

Reply:
https://github.com/dimagi/commcare-hq/pull/33735/commits/1f79cf5f35397123ac60fcd3ff6052c354cc65c5

Reply:
Thank you.